app/dashboard/views.py

```diff
@@ -300,8 +322,8 @@ def send_tip_2(request):
         'class': 'send2',
         'title': 'Send Tip',
         'recommend_gas_price': recommend_min_gas_price_to_confirm_in_time(confirm_time_minutes_target),
-        'from_email': primary_email,
-        'from_handle': username
+        'from_email': primary_from_email,
```

Comment:
thx for updating these variable names.. i can see why they were confusing in retrospect